src/api/test/functional/source_controller_test.rb

```diff
@@ -2009,7 +2009,7 @@ def test_remove_and_undelete_operations
 
     login_king
     post '/source/kde4', params: { cmd: :undelete }
-    assert_response :success
+    assert_response 200
```

Comment:
just wondering, why is this needed? :success should be 200, no?

Reply:
The `:success` accept any of the successful responses (200–299).
This test passed when we got `204 No content` code, but we really wanted a 200. So I changed it to 200 to be more specific.

Anyway, it's true that I can just leave the `:success`. Do you prefer that?

Reply:
Well, changed to `:success`.  We know it's due to a 200 and not a 204 code.